plugins/aggregators/starlark/starlark.go

```diff
@@ -0,0 +1,137 @@
+package starlark //nolint
```

Comment:
Needed to avoid getting `import-shadowing: The name 'starlark' shadows an import name (revive)`